boards/arm/mimxrt1170_evk/Kconfig.defconfig

```diff
@@ -33,6 +33,19 @@ endchoice
 
 endif #FLASH
 
+config I2C
+	default y if (KSCAN || SENSOR)
+
+if KSCAN
+
+config KSCAN_GT911
```

Comment:
remove `KSCAN_GT911`, will get enabled by Kconfig and devicetree association.

Reply:
Done